### PR TITLE
refactor: remove OGA backend service and clean up configurations

### PR DESCRIPTION
**Pull request by mushrafmim**
## Summary

This PR removes the deprecated `oga` (Other Government Agency) backend service from the `nsw` repository. This code has been extracted to its own dedicated `nsw-agency` repository to reduce coupled codebase dependencies and optimize CI/CD pipelines.

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update
- [x] Refactoring (no functional changes)
- [ ] Performance improvement
- [ ] Other (please describe):

## Changes Made

- Deleted the `oga` directory containing the OGA Go backend service codebase, tests, configurations, database connectors, and handlers.
- Updated `README.md` to remove `oga/Dockerfile` and associated build validation triggers.
- Updated `docs/CONTAINER_IMAGES.md`, `docs/DEPLOYMENT.md`, and `docs/RELEASE_IMAGES.md` to remove references to the OGA backend container images, local build instructions, and Kubernetes configuration guides.
- Updated `backend/pkg/blobsource/README.md` default local source path example from `/etc/oga/blobs` to `/etc/agency/blobs`.

## Testing

- [x] I have tested this change locally
- [ ] I have added unit tests for new functionality
- [ ] I have tested edge cases
- [x] All existing tests pass

## Checklist

- [x] My code follows the project's style guidelines
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have checked that there are no merge conflicts

## Related Issues

Related to #549

## Screenshots/Demo

N/A

## Additional Notes

None.

## Deployment Notes

N/A

### CODE REVIEW

File: docs/CONTAINER_IMAGES.md [hunk 1]
```diff
@@ -80,27 +77,9 @@ docker run --rm --name nsw-backend \
...
-  -e OGA_NSW_TOKEN_URL=https://localhost:8090/oauth2/token \
-  -e OGA_NSW_SCOPES= \
-  -e OGA_NSW_TOKEN_INSECURE_SKIP_VERIFY=true \
-  nsw-oga-backend:local
-```
-
 To run multiple OGA instances, start multiple containers with different names/ports and separate volumes.
 
-## 4.3 Trader Portal
+## 4.2 Trader Portal
```
[the remark this answers is not in the document]
Comment (mushrafmim) [reply docs/CONTAINER_IMAGES.md:82]: Addressed.

**Review by sthanikan2000 — APPROVED** [review]

**Review by ginaxu1 — APPROVED** [review]
